openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -63,6 +67,15 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
     client = oauth_adapter.get_client_for_token(access_token)
 
     jwt_expires_in = _get_jwt_access_token_expire_seconds()
+    try:
+        grant_type = access_token.application.authorization_grant_type
+    except Exception:
+        grant_type = ''
+        log.exception('Unable to get grant_type from access token.')
+
+    # .. custom_attribute_name: create_jwt_grant_type
+    # .. custom_attribute_description: The grant type of the newly created JWT.
+    set_custom_attribute('create_jwt_grant_type', grant_type)
```

Comment:
@jawad-khan: [inform] I added a custom attribute in case we want to review this data in New Relic.